apps/examples/wc-next10/package.json

```diff
@@ -31,5 +31,11 @@
   },
   "installConfig": {
     "hoistingLimits": "workspaces"
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-logical-assignment-operators": "7.22.11",
+    "@babel/plugin-transform-nullish-coalescing-operator": "7.22.11",
+    "@babel/plugin-transform-optional-chaining": "7.23.0",
+    "babel-loader": "8"
```

Comment:
@siggerzz was that meant to be 8.0.0 or it's valid to specify a version like this 👀